.classpath

```diff
@@ -1,28 +1,31 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should not include .classpath in GitHub. I've added it to .gitignore, add it too.